contracts/interfaces/draft-IERC7579.sol

```diff
@@ -63,6 +69,12 @@ interface IERC7579Validator is IERC7579Module {
     ) external view returns (bytes4);
 }
 
+/**
+ * @dev ERC-7579 Executor.
```

Comment:
This is IERC7579Hook. 

Hooks module are type id 4 and Execution modules are type id 2. So I'm guessing there is a confusion here

```suggestion
 * @dev ERC-7579 Hooks module.
```